pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/dashboard/DashboardLabelDto.java

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.dsm.db.dto.dashboard;
+
+import java.util.Optional;
+
+import lombok.Data;
+import org.apache.commons.lang3.StringUtils;
+
+@Data
+public class DashboardLabelDto {
+
+    Integer labelId;
+    Integer dashboardId;
+    String labelName;
+    String color;
+    DashboardLabelFilterDto dashboardFilterDto;
+
+    private DashboardLabelDto(Builder builder) {
+        this.labelId = builder.labelId;
+        this.dashboardId = builder.dashboardId;
+        this.labelName = builder.labelName;
+        this.color = builder.color;
+        this.dashboardFilterDto = builder.dashboardLabelFilterDto;
+    }
+
+    public String getEsNestedPath() {
+        return Optional.ofNullable(dashboardFilterDto).map(DashboardLabelFilterDto::getEsNestedPath).orElse(StringUtils.EMPTY);
+    }
+
+    public static class Builder {
+
```

Comment:
same question regarding lomboks @Builder

Reply:
Agree -- this is a question for @SimoneMaiwald   -- both builders and lombok @Setter are valid approaches, but we should standardize on one approach going forward.  Simone -- which style should we use?

Reply:
Now that we are in 1 repo and DSS and DSM are moving closer together we should make that a DDP decision! Maybe next "Pepper Process & Standards" should be used to make a decision on that.